test/multicall/Multicall.js

```diff
@@ -266,8 +264,9 @@ describe('Multicall Test Suite', function () {
       expect(bytes).to.gte(42624);
     });
 
-    it('should NOT be able to aggregate 100 calls to processLongOutput', async function () {
-      const n = 100;
+    it('should NOT be able to aggregate 115 calls to processLongOutput', async function () {
+      const n = 115;
+      const maxDataSize = 25 * 1024 * 2; // 25 kb
```

Comment:
Q: this is 50KBs, not 25Kbs like the comment says right?

Reply:
```java
    private void validateContractData(final ContractCallRequest request) {
        var data = request.getData();
        if (data != null
                && !evmProperties.getDataValidatorPattern().matcher(data).find()) {
            throw new InvalidParametersException(
                    "data field of size %d contains invalid hexadecimal characters or exceeds %d characters"
                            .formatted(
                                    data.length(),
                                    evmProperties.getMaxDataSize().toBytes() * 2L));
        }
    }
```

Basically 2 hex characters represent 1 byte => maxDataSize is 25kb, which is 25 * 1024 * 2 in hexadecimal characters. Maybe the comment is a bit confusing, sorry.

Reply:
Ahh I see so it indicates the length of the hex string. It makes sense now! Thanks for the clarification!